python-cache-demo/stats-service/Dockerfile

```diff
@@ -11,4 +11,4 @@ COPY . .
 
 ENV FLASK_APP=stats_service.py
 
-CMD [ "python3", "-m" , "flask", "run", "--host=0.0.0.0"]
+CMD [ "python3", "-u", "-m" , "flask", "run", "--host=0.0.0.0"]
```

Comment:
Does unbuffering the stdout and stderr help us? Was this a problem? (I'm genuinely curious, haven't had troubles with this yet)

Reply:
When testing and tackling different issues it helped me with debugging, but indeed "in production" or once everything works it will not be required. My bad for not removing it and thanks for spotting!
Already deleted the flag.

Reply:
Great to know! Don't see this one often :)